packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/resetSimulation.st

```diff
@@ -1,5 +1,15 @@
 simulation
 resetSimulation
 	
-	self stopSimulation.
-	self simulationTime: 0.
\ No newline at end of file
+	"remove al submorph except the first ever added -> start reset Buttons"
+	self nodes size timesRepeat: [ (self nodes at: 1) delete]. 
+	
+	self resetState do:[:morphy |
+							self addMorph: morphy].
+								
+								
+	"Since i don't know how to redraw the morph, we have to stick with this. I know it's ugly. BUT IT WORKS"
+	self position: (self position + (1@1)).
+	self position: (self position - (1@1)).
```

Comment:
this does not look good, but helps solving the drawing process. Anyone a better idea?